src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -62,11 +62,11 @@ export const TradeConfirm = ({ history }: RouterProps) => {
 
   // Parametrized errors cannot simply be matched with === since their param(s) might vary
   const PARAMETRIZED_ERRORS_TO_TRADE_ERRORS = {
-    'ZrxExecuteQuote - signAndBroadcastTransaction error': TRADE_ERRORS.TRANSACTION_REJECTED,
-    'ZrxExecuteQuote - Signed transaction is required': TRADE_ERRORS.SIGNING_REQUIRED,
-    'ZrxExecuteQuote - broadcastTransaction error': TRADE_ERRORS.BROADCAST_FAILED,
-    'ZrxExecuteQuote - invalid HDWallet config': TRADE_ERRORS.HDWALLET_INVALID_CONFIG,
-    'ZrxExecuteQuote - signTransaction error': TRADE_ERRORS.SIGNING_FAILED,
+    'ZrxExecuteTrade - signAndBroadcastTransaction error': TRADE_ERRORS.TRANSACTION_REJECTED,
+    'ZrxExecuteTrade - Signed transaction is required': TRADE_ERRORS.SIGNING_REQUIRED,
+    'ZrxExecuteTrade - broadcastTransaction error': TRADE_ERRORS.BROADCAST_FAILED,
+    'ZrxExecuteTrade - invalid HDWallet config': TRADE_ERRORS.HDWALLET_INVALID_CONFIG,
+    'ZrxExecuteTrade - signTransaction error': TRADE_ERRORS.SIGNING_FAILED,
```

Comment:
Should this be an enum, so we only have to change these strings in one place?

Reply:
It can be a separate PR, it's not worth holding up this PR - I'll merge.

Reply:
Yeah agreed enums would have been way cleaner, however I got some missing bit of context from @elmutt that this will very soon disappear for better error-handling following the swapper refactor so it's fine to keep it as is for a few more days!